Update GitLab and Gitlab-v2 integration slugs (#2193)
port-labs/port-docs: docs/build-your-software-catalog/sync-data-to-catalog/git/gitlab-v2/GitLab-v2.md

# GitLab

> :::info

Reviewer: ```suggestion
:::info GitLab v2 documentation
```

> You’re viewing the latest GitLab integration, released in April 2025. For details on the previous version, check out the [GitLab (deprecated)](/build-your-software-catalog/sync-data-to-catalog/git/gitlab/) documentation.  

Reviewer: ```suggestion
This page documents the latest GitLab integration, released in April 2025.  
For documentation of the previous integration, check out the [GitLab (deprecated)](/build-your-software-catalog/sync-data-to-catalog/git/gitlab/) page.  
```
